src/crimson/tools/store_nbd/fs_driver.h

```diff
@@ -6,16 +6,10 @@
 #include "crimson/os/futurized_collection.h"
 #include "crimson/os/futurized_store.h"
 
-namespace seastar::alien {
-class instance;
-}
-
 class FSDriver final : public BlockDriver {
 public:
-  FSDriver(config_t config,
-	   seastar::alien::instance& alien)
-  : config(config),
-    alien(alien)
```

Comment:
sorry for breaking the build. and thanks for fixing it.

might want to update `FSDriver::init()` accordingly?